modules/ROOT/pages/bloom-installation/bloom-prerequisites.adoc

```diff
@@ -45,7 +45,7 @@ image::compatibility-mode.png[width=400,align="center"]
 
 [NOTE]
 --
-To obtain activation keys for the Bloom client or server components, ask your friendly neighborhood Neo4j representative.
+You obtain activation keys for the Bloom Enterprise plugin from your Neo4j representative.
```

Comment:
not related to this line,  in line 36 there is an image of settings which include the classic search, should we remove it or maybe we should put in a list for doing a batch updates of images?
![Screenshot 2025-02-25 at 16 52 09](https://github.com/user-attachments/assets/6efa2e9b-ea14-4c7d-b939-49d9d14c270b)